commands/music/musictrivia.js

```diff
@@ -11,7 +11,7 @@ module.exports = class MusicTriviaCommand extends Command {
       memberName: 'music-trivia',
       aliases: ['music-quiz', 'start-quiz'],
       group: 'music',
-      description: 'Engage in a music quiz with your friends! -Beta Feature!',
+      description: 'Engage in a music quiz with your friends!',
       guildOnly: true,

```

Comment:
it kept failing when I had the `default:5` wanted to denoted it may be unstable with "- Beta Feature" wording but has been since fixed and was removed

Reply:
If it was fixed why did you uncomment the default property?

Reply:
because for some reason with `defualt:5` for me it was would never finish a round but when it was removed in Favor to ask how man it worked without Hang ups

Reply:
it was the simplest solution i could think of yesterday

Much Love
-Bacon

Reply:
I'll look into it now to see if I'm getting it aswell

Reply:
yeah it happens to me aswell, very weird